src/Symfony/Component/Console/Tests/Command/InvokableCommandTest.php

```diff
@@ -27,6 +27,8 @@
 
 class InvokableCommandTest extends TestCase
 {
+    private const FUNCTION_NAME = 'Symfony\Component\Console\Tests\Command\InvokableCommandTest::Symfony\Component\Console\Tests\Command\{closure}()';
```

Comment:
I don't understand the 2 namespaces? It should be `Symfony\Component\Console\Tests\Command\{closure}()`

Reply:
I copied the code to extract the function name from https://github.com/symfony/symfony/blob/cb084808343a10152d4ec69c658f75878d41d3d3/src/Symfony/Component/DependencyInjection/Attribute/Target.php#L55. With closures it's a little wonky I admin - plus it changes in 8.4 to include more context (hence the current failures).

For invokable classes, which most of these new commands will be in practice, the name looks like you expect: `App\Command\SendSalesReportsCommand::__invoke()`